rpc/handlers_test.go

```diff
@@ -1492,7 +1501,7 @@ func TestEvents(t *testing.T) {
 	t.Run("filter with no keys", func(t *testing.T) {
 		var allEvents []*rpc.EmittedEvent
 		t.Run("get all events without pagination", func(t *testing.T) {
-			args.ToBlock = &rpc.BlockID{Latest: true}
+			args.ToBlock = &rpc.BlockID{Pending: true}
```

Comment:
We should create a new test case, instead of overwriting a previous one, right?

Reply:
testcase is "get all events without pagination", "all events used" to mean "0 to Latest", but now "0 to Pending" because we can possibly have a pending block and matching events in the pending block now.

Reply:
No need to add an extra case for this as your explanation makes sense